vendor/bat-native-ads/src/bat/ads/internal/rpill/rpill_helper.cc

```diff
@@ -0,0 +1,75 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/rpill/rpill_helper.h"
+
+#include <string>
+#include <vector>
+
+#include "base/strings/string_util.h"
+#include "bat/ads/ads.h"
+
+namespace ads {
+
+namespace {
+
+std::string GetHardware() {
+  const std::string manufacturer = base::ToLowerASCII(_sys_info.manufacturer);
```

Comment:
this is an example of why these are prefixed with `g_` because otherwise it's not clear where `_sys_info` came from

Reply:
Agreed and will replace all 4 of these in this PR with `g_`